packages/client/package.json

```diff
@@ -11,6 +11,7 @@
     "react-ga": "^2.7.0",
     "react-router-dom": "^5.1.2",
     "react-signature-canvas": "^1.0.3",
+    "react-toastify": "^6.0.5",
```

Comment:
How many KB's does this package contribute to the deploy size?

Reply:
@tianhuil according to Bundlephobia, about 7kb when minified.

https://bundlephobia.com/result?p=react-toastify@6.0.5